scripts/initializeProposal.ts

```diff
@@ -11,9 +12,26 @@ anchor.setProvider(provider);
 
 const payer = provider.wallet["payer"];
 
-const PANTERA_PUBKEY = new PublicKey("BtNPTBX1XkFCwazDJ6ZkK3hcUsomm1RPcfmtUrP6wd2K");
+const FAAS_SQUADS_MULTISIG = new PublicKey("AHwsoL97vXFdvckVZdXw9rrvnUDcPANCLVQzJan9srWy");
+
+const COST_DEPLOY = 4 * LAMPORTS_PER_SOL
 
 async function main() {
+  console.log('Initializing with PublicKey');
+  console.log(payer.publicKey.toString());
+
+  const payerBalance = await provider.connection.getBalance(payer.publicKey);
+  console.log('Account SOL balance');
+  console.log(payerBalance / LAMPORTS_PER_SOL);
+  if (payerBalance < COST_DEPLOY){
```

Comment:
isn't `COST_DEPLOY` represented as lamports because it's `4*LAMPORTS_PER_SOL`? But I could be totally missing something here. But then it looks like we are comparing SOL to lamports here.

Reply:
nope nvm. Still waking up

Reply:
Not sure what you're getting at. This looks correct. Everything is represented in terms of lamports (9 decimals).

Reply:
yeah I was looking at it incorrectly. Hence my updated comment